### PR TITLE
feat(initial): release initial helm-chart to start community work

### PR DESCRIPTION
**Pull request by haarchri**
Signed-off-by: haarchri <chhaar30@googlemail.com>

feat(initial): release initial helm-chart to start community work

`helm install ./charts/flux2 --generate-name --create-namespace=true --namespace=flux `

```
kubectl get pods -n flux
flux          helm-controller-7dbc5b5674-s27qc                                  1/1     Running   0          2m21s
flux          image-automation-controller-bb85b9689-rmllt                       1/1     Running   0          2m21s
flux          image-reflector-controller-f8b7bfdc7-xkth6                        1/1     Running   0          2m21s
flux          kustomize-controller-5bcdddf8d6-6d92h                             1/1     Running   0          2m21s
flux          notification-controller-6f85d55dc8-2h4gw                          1/1     Running   0          2m21s
flux          source-controller-7bc4d985c6-98r5q                                1/1     Running   0          2m21s
```

### CODE REVIEW

File: charts/flux2/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# flux2
+
+![Version: 0.1.1](https://img.shields.io/badge/Version-0.1.1-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.20.1](https://img.shields.io/badge/AppVersion-0.20.1-informational?style=flat-square)
```
Comment (pierluigilenoci) [inline charts/flux2/README.md:3]: 0.1.1?

File: charts/flux2/templates/helm-controller.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+{{- if and .Values.helmcontroller.create}}
+apiVersion: v1
```
Comment (pierluigilenoci) [inline charts/flux2/templates/helm-controller.yaml:2]: can they be separated to have a single manifest template?

File: charts/flux2/templates/crds/crds-helm-controller.yaml [hunk 3]
```diff
@@ -0,0 +1,757 @@
+{{- if and .Values.helmcontroller.create}}
```
Comment (pierluigilenoci) [inline charts/flux2/templates/crds/crds-helm-controller.yaml:1]: for now it is ok but in the future I will like to have an option like this https://github.com/jetstack/cert-manager/blob/master/deploy/charts/cert-manager/values.yaml#L42

File: charts/flux2/templates/policies.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
+{{- if and .Values.policies.create}}
+apiVersion: networking.k8s.io/v1
```
Comment (pierluigilenoci) [inline charts/flux2/templates/policies.yaml:2]: Aren't they a little too open?

**Review by pierluigilenoci — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# Helm Charts
+
+The repository can be added using the following command:
+```bash
+helm repo add haarchri https://fluxcd-community.github.io/helm-charts
```
[the remark this answers is not in the document]
Comment (haarchri) [reply README.md]: done

File: charts/flux2/README.md (hunk 1 above)
Comment (haarchri) [reply charts/flux2/README.md]: done

File: charts/flux2/templates/crds/crds-helm-controller.yaml (hunk 3 above)
Comment (haarchri) [reply charts/flux2/templates/crds/crds-helm-controller.yaml:1]: good question here is if the controller pods can run without the crds - lets discuss this via an an issue and a follow up PR - thank

File: charts/flux2/templates/helm-controller.yaml (hunk 2 above)
Comment (haarchri) [reply charts/flux2/templates/helm-controller.yaml:2]: yes lets do it in a followup PR - thanks

File: charts/flux2/Chart.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: flux2
+description: A Helm chart for flux2
+
+type: application
+version: 0.1.1
```
[the remark this answers is not in the document]
Comment (haarchri) [reply charts/flux2/Chart.yaml]: done

**Comment by haarchri** [comment]
@pierluigilenoci feel free to merge ;)

**Pull request merged**